app/services/recovery_code_generator.rb

```diff
@@ -10,12 +10,33 @@ def create
     raw_recovery_code
   end
 
+  def verify(plaintext_code)
+    uak = user_access_key(plaintext_code)
+    encryption_key, encrypted_code = user.recovery_code.split(Pii::Encryptor::DELIMITER)
+    begin
+      key_maker.unlock(uak, encryption_key)
+    rescue Pii::EncryptionError => _err
+      return false
```

Comment:
Would it be worth logging the exception here to (either to the logger or to NewRelic?) or do we expect these failures might accidentally log the payload or key?